project_event/wizard/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding:utf-8 -*-
```

Comment:
@rimbendhaou :  Same comment as above. It is not required in 11.0 "-*- coding:utf-8 -*-"